akka-http2-support/src/main/scala/akka/http/scaladsl/model/http2/Http2SettingsHeader.scala

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.scaladsl.model.http2
+
+import akka.annotation.InternalApi
+import akka.http.scaladsl.model.headers.{ CustomHeader, ModeledCustomHeader, ModeledCustomHeaderCompanion, RequestHeader }
+import akka.http.impl.engine.http2.framing.Http2FrameParsing
+import akka.http.impl.engine.http2.FrameEvent.Setting
+import akka.http.impl.model.parser.Base64Parsing
+import akka.stream.impl.io
+import akka.stream.impl.io.ByteStringParser
+import akka.util.ByteString
+
+import scala.collection.immutable
+import scala.util.Try
+
+/**
+ * Internal API
+ */
+@InternalApi object Http2SettingsHeader {
```

Comment:
`private[akka]` I know it gets boring with all the markers... ;-) Eventually we'll get the gains from the annotation tho!